src/components/ArticlesById.jsx

```diff
@@ -16,6 +19,17 @@ useEffect(() => {
       .catch((err) => {
       });
   }, [article_id]);
+
+  function handleVote(e) {
+    e.preventDefault()
+      setVote(vote + 1)
+      console.log(vote)
+    postVote(newVote, article_id).then(()=>{
+        window.location.reload()
+    }).catch((err)=> {
```

Comment:
The postVote function is not an appropriate name as we are not posting anything here, were are updating the article with a patch request, I would recommend renaming this function to something more appropriate to what it is actually doing.

This is also not the ideal way to update the webpage with the new vote. 
You have a state that you are trying to update here and also use directly after which is not something that will work as setting state is an asynchronous operation, so it will not update fast enough for you to use its updated value straight away. 
Instead of using state here you can manually give the value of what should be sent to this patch request, and maybe consider using state to show that something has happened to the user instead of forcing a refresh like this.